sdl-core/pom.xml

```diff
@@ -111,19 +111,17 @@
             <scope>${spark.deps.scope}</scope>
         </dependency>
 
-        <!-- this needs to have compile scope, if it's runtime it doesnt compile anymore -->
```

Comment:
hmm this comment is not true anymre?

Reply:
now scope is compile or provided, both are ok. I'll improve the comment.